content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js

```diff
@@ -118,6 +108,27 @@
             }
         }
 
+        /**
+         * Displays the panel containing the element that corresponds to the deep link in the URI fragment
+         * and scrolls the browser to this element.
+         */
+        function scrollToDeepLinkIdInTabs() {
+            if (containerUtils) {
+                var deepLinkItemIdx = containerUtils.getDeepLinkItemIdx(that, "tab", "tabpanel");
+                if (deepLinkItemIdx && deepLinkItemIdx !== -1) {
+                    var deepLinkItem = that._elements["tab"][deepLinkItemIdx];
+                    if (deepLinkItem && that._elements["tab"][that._active].id !== deepLinkItem.id) {
+                        navigateAndFocusTab(deepLinkItemIdx, true);
+                    }
+                    var hashId = window.location.hash.substring(1);
+                    var hashItem = document.querySelector("[id='" + hashId + "']");
```

Comment:
See comment above regarding empty hashId

Reply:
Addressed in https://github.com/adobe/aem-core-wcm-components/pull/2138/commits/5b6735af3ae0bf75e885c69d9bf51dc94cd643be